lib/MongoDB/Op/_Aggregate.pm

```diff
@@ -121,6 +120,12 @@ sub execute {
 
     my $has_out = $self->has_out;
 
+    if ( $link->server->is_master->{maxWireVersion} < 6
+      && $self->coll_name eq 1 ) {
+        MongoDB::Error->throw(
+            "Cannot use MongoDB::Database::_aggregate (db.aggregate) function on Wire Version < 6" );
```

Comment:
I'd rather not have the outer API method name here in the error message. Better would be a message like "Calling aggregate with a collection name of '1' is not supported on Wire Version < 6'

Reply:
Ok, have changed to that error message. Wasnt sure what to put there in the first place heh